cmd/main.go

```diff
@@ -116,47 +127,50 @@ func main() {
 		fmt.Fprintf(os.Stderr, "Unabled to parse z: %v", err)
 		usage()
 	}
-	x, err := strconv.ParseUint(parts[1], 10, 64)
+	x, err := strconv.ParseInt(parts[1], 10, 64)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Unabled to parse x: %v", err)
 		usage()
 	}
-	y, err := strconv.ParseUint(parts[2], 10, 64)
+	y, err := strconv.ParseInt(parts[2], 10, 64)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "Unabled to parse y: %v", err)
 		usage()
 	}
-	tile := slippy.NewTile(uint(z), uint(x), uint(y))
+	tile := slippy.Tile{
+		Z: slippy.Zoom(z),
+		X: uint(x),
+		Y: uint(y),
```

Comment:
if you use `ParseUint` above this cast is no longer needed.

Reply:
They are still need as ParseUint returns a uint64 not a uint. But am making the changes above.

Reply:
ok, but I think if we ran that through a code scanner we would get an overflow warning.